libact/query_strategies/multiclass/mdsp.py

```diff
@@ -16,8 +16,8 @@
 from sklearn.base import BaseEstimator
 from sklearn.metrics import euclidean_distances
 from sklearn.utils import check_random_state, check_array, check_symmetric
-from sklearn.externals.joblib import Parallel
-from sklearn.externals.joblib import delayed
+from joblib import Parallel
+from joblib import delayed

```

Comment:
```suggestion
from joblib import Parallel, delayed

```

nit, after this, please go ahead and merge it.
Thanks.

Reply:
I've migrated the CI to github action.
Please rebase your PR and check the CI result.
As long as on python 2.7 3.4 3.5 3.6 3.7 3.8 on ubuntu is passing, you can go ahead and merge.